src/rbus/rbus.c

```diff
@@ -2689,8 +2689,8 @@ rbusError_t rbus_get(rbusHandle_t handle, char const* name, rbusValue_t* value)
                 errorcode = CCSPError_to_rbusError(legacyRetCode);
             }
         }
-
-        rbusMessage_Release(response);
+        if(response != NULL)//check if "response" is already Freed And Set to Null in rbus_invokeRemoteMethod()
```

Comment:
this check may not be needed as it is already freed by rbusMessage_Release() function per recent change.

Reply:
Removed this if(response !=NULL) as per your review comment. Changes are committed as part of commit a9e1fdaf3edf419bb45e7b3470ed54224b04aeb6.